examples/word-count/dev/user.clj

```diff
@@ -0,0 +1,118 @@
+(ns user
+  (:require [clojure.java.shell :refer [sh]]
+            [clojure.string :as str]
+            [jackdaw.admin :as ja]
+            [jackdaw.repl :refer :all]
+            [jackdaw.serdes :as js]
+            [jackdaw.streams :as j]
+            [jackdaw.streams.describe :as jsd]
+            [integrant.core :as ig]
+            [integrant.repl :refer [clear go halt prep init reset reset-all]]
+            [word-count])
+  (:import [clojure.lang ILookup Associative]))
+
+
+(def config
+  {:streams-config {:application-id "word-count"
+                    :bootstrap-servers "localhost:9092"
+                    :default-key-serde "jackdaw.serdes.EdnSerde"
+                    :default-value-serde "jackdaw.serdes.EdnSerde"
+                    :cache-max-bytes-buffering "0"}
+
+   :topology {:topology-builder-fn :word-count/topology-builder}
+
+   :topics {:streams-config (ig/ref :streams-config)
+            :topology (ig/ref :topology)}
+
+   :app {:streams-config (ig/ref :streams-config)
+         :topology (ig/ref :topology)
+         :topics (ig/ref :topics)}})
+
+
+(integrant.repl/set-prep! (constantly config))
+
+
+(deftype FakeTopicMetadata []
+  ILookup
+  (valAt [this key]
+    {:topic-name (name key)
+     :partition-count 1
+     :replication-factor 1
+     :key-serde (js/edn-serde)
+     :value-serde (js/edn-serde)})
+
+  Associative
+  (assoc [this key val]
+    this))
+
+(defn new-fake-topic-metadata []
```

Comment:
I think we can just use a single instance of `FakeTopicMetadata`, so we can drop this function and update `topic-metadata` to be:
```clojure
(def topic-metadata (FakeTopicMetadata.))
```
Then we can just use `topic-metadata` where `new-fake-topic-metadata` is currently used.

Reply:
Could do. Does this still make sense if we add an arity to allow the user to set their own default?